server/container/guice/cassandra-guice/src/test/java/org/apache/james/DockerCassandraRule.java

```diff
@@ -48,7 +48,7 @@ public PropertiesConfiguration getCassandraConfigurationForDocker() {
         configuration.addProperty("cassandra.keyspace", "apache_james");
         configuration.addProperty("cassandra.replication.factor", 1);
         configuration.addProperty("cassandra.retryConnection.maxRetries", 10);
-        configuration.addProperty("cassandra.retryConnection", 5000);
+        configuration.addProperty("cassandra.retryConnection.minDelay", 5000);
```

Comment:
Good catch!

Please we should be 100% compatible with old config...

Consider checking **cassandra.retryConnection** if there is no **cassandra.retryConnection.minDelay** (and of course have a default value as a fallback)